spec/unit/frame_spec.rb

```diff
@@ -112,6 +112,31 @@
       "└─────────────────────────────────┘\n"
     ].join)
   end
+  
+  it "correctly envelopes colored text" do
+  	box = TTY::Box.frame do
+      Pastel.new.green.on_red("Hello world!")
+    end
+    
+    expect(box).to eq([
+    "┌────────────┐\n",
+	"│\e[32;41mHello world!\e[0m│\n",
+	"└────────────┘\n"
+	].join)
+  end
+  
+  it "correctly spaces colored titles" do
+  	p = Pastel.new
+  	box = TTY::Box.frame(title: {top_left: p.green.on_red("TITLE"), bottom_right: p.green.on_red("(v1.0)")}) do
```

Comment:
Maybe wrap the `:top_left` and `:bottom_right` keys into multiple lines to break up the long line?